src/Sylius/Bundle/CurrencyBundle/Templating/Helper/CurrencyHelperInterface.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CurrencyBundle\Templating\Helper;
+
+use Sylius\Bundle\MoneyBundle\Templating\Helper\MoneyHelperInterface;
+use Sylius\Component\Currency\Context\CurrencyContextInterface;
+use Sylius\Component\Currency\Converter\CurrencyConverterInterface;
+use Symfony\Component\Templating\Helper\Helper;
+
+/**
+ * @author Axel Vankrunkelsven <axel@digilabs.be>
+ */
+interface CurrencyHelperInterface
+{
+    /**
+     * @param CurrencyContextInterface $currencyContext
+     * @param CurrencyConverterInterface $converter
+     * @param MoneyHelperInterface $moneyHelper
+     */
+    public function __construct(CurrencyContextInterface $currencyContext, CurrencyConverterInterface $converter, MoneyHelperInterface $moneyHelper);
+
+    /**
+     * Convert amount to target or currently used currency.
+     *
+     * @param int $amount
+     * @param string|null $currency
+     *
+     * @return string
+     */
+    public function convertAmount($amount, $currency = null);
+
+    /**
+     * Convert amount and format it!
+     *
```

Comment:
I guess we could get rid of this comment since the method name already says this.